Added support for :help vim-pencil

iandstanley opened the pull request:
Created sub-directory doc and added vim-pencil.txt containing the documentation from the README

This allows the user to run 

> :help vim-pencil 

To get a hypertext help in the same manner as all help is available

iandstanley commented:
Many people like using the internal vim help system so I ported the README into the vim help format (see doc/vim-pencil.txt)

iandstanley commented:
There’s almost everything in the help

Except the see also bits 

> On 29 Jul 2020, at 17:29, Caleb Maclennan <notifications@github.com> wrote:
> 
> ﻿
> @alerque approved this pull request.
> 
> Fantastic, thanks for taking the trouble to do this. I might even suggest following up to make the README even more concise and skip things that non-users of the plugin don't need to review in detail and direct them to :help for full details. We should still cover the features so people can understand what is available and whether they want to use the plugin, but some of the nitty gritty could be dealt with in help better that here.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.


-- Email domain proudly hosted at https://migadu.com

alerque commented on doc/vim-pencil.txt:
```diff
@@ -0,0 +1,651 @@
+
+            Rethinking Vim as a tool for writers 
+
+CONTENTS                                                             *vimwiki*
```
Vimwiki? This looks like a copy/paste leftover from something you used as a template perhaps?

iandstanley commented:
updated reference to vimwiki

Yeah I took the vimwiki/doc/vimwiki.txt as a starting point and replaced all text 

I mistakenly started with 1. Intro and missed the vimwiki reference 

Replaced it with vim-pencil
